listenbrainz/listenstore/redis_listenstore.py

```diff
@@ -30,7 +30,7 @@ def get_playing_now(self, user_id):
         if not data:
             return None
         data = ujson.loads(data)
-        data.update({'listened_at': MIN_ID+1})
+        data.update({'listened_at': None})
```

Comment:
Instead of directly putting None here, I would prefer it if we added a `playing_now` key to the data and then set the timestamp attribute in the from_json function there accordingly.